README.md

```diff
@@ -37,6 +55,55 @@ $userConsent->setUsage('E-mail marketing campaigns');
 $userConsent->setUsageFieldName('gdpr_consent_usage');
 
 $subscriber->subscribe('hello@super.test', ['Name' => 'John Doe'], [$userConsent]);
+
+/**
+ * Method for sending transactional emails (YMLP does not support transactional emails).
+ *
+ * @param array<\SubscribeMe\ValueObject\EmailAddress> $emails (email required, name optional)
+ * @param int|string $emailTemplateId required
+ * @param array $variables optional
+ */
+$subscriber->sendTransactionalEmail($emails, $emailTemplateId, $variables)
+```
+
+### SYMFONY Implementation
+
+With symfony, you don't have to implement the factory,
+you can directly implement the interface and thus make the code generic,
+which means that if you want to change platform later,
+you just have to change the register implementation.
+
+```php
+class YourClass
+{
+    public function __construct(
+        private SubscriberInterface $subscriber
+    ) {
+    }
+    
+    public function sendTransactional()
+    {
+        $this->subscriber->sendTransactionalEmail(
+            [
+                new EmailAddress('user@example.com')
+            ],
+            $templateId
+        )
+    }
+}
+```
+
+```yaml
+# services.yaml
+services:
+  SubscribeMe\Subscriber\SubscriberInterface:
+    # here call the class of the platform used in your project (example with mailjet)
+    class: SubscribeMe\Subscriber\MailjetSubscriber
+    autowire: true
+    calls:
+      # here call method necessary on the platform (mailjet need apiKey and apiSecret)
+      - setApiKey: [ '%env(string:APP_MAILJET_API_KEY)%' ]
+      - setApiSecret: [ '%env(string:APP_MAILJET_API_SECRET_KEY)%' ]
 ```
```

Comment:
```suggestion
```yaml
# services.yaml
services:
  SubscribeMe\Subscriber\SubscriberInterface:
    # Here register the platform class used in your project (example with Mailjet)
    class: SubscribeMe\Subscriber\MailjetSubscriber
    # Here comes the Symfony magic, PSR17 and PSR18 will be automatically provided
    autowire: true
    calls:
      # Here call necessary methods according to your platform (Mailjet need apiKey and apiSecret)
      - setApiKey: [ '%env(string:APP_MAILJET_API_KEY)%' ]
      - setApiSecret: [ '%env(string:APP_MAILJET_API_SECRET_KEY)%' ]
```
```